docs/source/AdministratorGuide/Systems/MonitoringSystem/index.rst

```diff
@@ -113,6 +113,9 @@ You can configure the MQ in the local dirac.cfg file where the agent is running:
      }
    }
 
+A dashboard for WMSHistory monitoring ``WMSDashboard.json`` is available for import as a JSON file `here <https://github.com/rubenpozzi97/DIRAC/tree/integration/dashboards>`__.
+The dashboard is compatible with ElasticSearch (not older versions such as ES6) and Opensearch visualization softwares such as Kibana or Opensearch Dashboards.
+To import it in the Kibana UI, go to Management -> Saved Objects -> Import and import the JSON file (you may need to adapt your index patterns).
```

Comment:
Will there be some substitutions to make?

Reply:
Do you mean regarding index pattern etc?

Reply:
Yes, but also field names.

Reply:
Not necessarily. For the field names, it depends on the kind of indices that are stored in the user's node. But given that the kind of data we're talking about is the same it should be fine right?
And regarding index patterns, if the system does not recognize the same index pattern when a user is importing the JSON file into the Kibana UI, they can automatically assign the imported dashboard and visualizations to one of the existing index patterns in the user's setup, so that should also be fine.